release-team/role-handbooks/release-notes/README.md

```diff
@@ -0,0 +1,116 @@
+# Kubernetes Release Team: Release Notes
+
+## Overview:
+
+The Release Notes role is responsible for collecting and fine-tuning release-notes from the many contributions to kubernetes between release cycles.
+This role is likely to find that work during the first several weeks of the release cycle is very laid back with the bulk of the tasks being
+completed at the end, once the release is firmed up.
+
+## Skills and Experience Required:
+
+* Strong written and verbal communications skills
+
+* A working knowledge of Kubernetes concepts
+
+* Project management experience is helpful
+
+## Tasks
+
+- The Release notes lead and shadows attend burn down meetings, sig-release meetings and follow the sig-release slack chat for relevant
+information throughout the release cycle to keep up with relevant release specific information.
+- One member of the release notes team should be responsible for setting up and running the [release notes tool](https://github.com/marpaia/release-notes)
+to collect generated drafts of all release-notes identified in the current release -- improvements to the tool are always welcome!
+- Drafts produced by the tool are copied to Hackmd and shared with the rest of the release team (there is currently no automated way to keep the generated
```

Comment:
We generally don't do this. Throughout the release cycle, we'll update the draft in this repo, but we never really use HackMD. We planned on using it for collaborating with SIG Leads, but as you know, the release notes wound up being too long and we didn't use it.

Reply:
Yes. I haven't updated this since that became apparent. Is the standard to use google docs then? I'm going to update this now.